README.md

```diff
@@ -19,3 +19,20 @@ To download, compile and configure Vim:
 ```
 ./do.sh
 ```
+
+# get-vim for Windows
+
+If decide use vim in Windows use the script 'do-win.sh'
+
+### Dependencies
+
+- [Cmdr](https://cmder.net/)
+- [Chocolatey](https://chocolatey.org/)
```

Comment:
How mainstream are `Cmdr` and `Chocolatey`?

Reply:
cmdr is the best terminal option for work in windows; exists alternatives but less complete.
chocolatey is software management automation like <<npm>>, brings the possibility to install applications with a simple command on terminal.